Packs/BloodHoundEnterprise/Integrations/BloodHoundEnterprise/BloodHoundEnterprise_test.py

```diff
@@ -0,0 +1,397 @@
+    )
+    assert all(
+        (
+            next_run.get("last_event_id") == 2057,
+            next_run.get("prev_fetch_id") == 1,
+            next_run.get("skip") == 7,
+        )
+    )
+    assert events[0].get("id") == 2051
+
+
+@freeze_time("2024-11-20T13:17:24.074375+02:00")
+def test_fetch_events_second_time(mocker):
+    """
+    Given:
+        - A mock client with a predefined set of test events loaded from a JSON file.
+        - The current date and time are frozen at "2024-11-20T13:17:24.074375+02:00".
+        - The previous fetch history is simulated using `demisto.getLastRun`, with the following values:
+            - `last_event_created_at` set to "2024-11-24T12:43:57.27948Z".
+            - `last_event_id` set to 2072.
+            - `prev_fetch_id` set to 1.
+
+    When:
+        - The `fetch_events` function is called to retrieve events with the mock client,
+            with a parameter for `max_events_per_fetch` set to "2".
+
+    Then:
+        - Ensure that exactly 2 events are fetched.
+        - Validate that the first event in the fetched list has the correct ID (2073).
+        - Verify that the next run metadata (`next_run`) is correctly updated with:
+            - `last_event_created_at` timestamp.
+            - `last_event_id` of the last fetched event (2074).
+            - `prev_fetch_id` incremented to 2.
+            - `skip` indicating the number of events fetched (2).
+    """
+    from BloodHoundEnterprise import fetch_events
+
+    mocker.patch.object(
+        demisto,
+        "getLastRun",
+        return_value={
+            "last_event_created_at": "2024-11-24T12:43:57.27948Z",
+            "last_event_id": 2072,
+            "prev_fetch_id": 1,
+        },
+    )
+    mocker.patch.object(demisto, "debug")
+    test_events = load_test_events()
+    client = create_mock_client(test_events)
+    next_run, events = fetch_events(
+        client=client,
+        params={"max_events_per_fetch": "2"},
+    )
+
+    assert len(events) == 2
+    assert events[0].get("id") == 2073
+    assert next_run == {
+        "last_event_created_at": "2024-11-24T12:43:57.27948Z",
+        "last_event_id": 2074,
+        "prev_fetch_id": 2,
+        "skip": 2,
+    }
+
+
+def test_test_module_command(mocker):
+
+    from BloodHoundEnterprise import test_module
+
+    """
+    Given:
+        - A mock client with a predefined set of test events loaded from a JSON file.
+        - The `_request` method of the mock client is patched to avoid making actual requests.
+
+    When:
+        - The `test_module` function is called to test the module's functionality with the mock client.
+
+    Then:
+        - Ensure that the result of the `test_module` function is "ok", indicating the module is working as expected.
+    """
+    test_events = load_test_events()
+    client = create_mock_client(test_events)
+    mocker.patch.object(client, "_request")
+    res = test_module(client)
+    assert res == "ok"
```

Comment:
No need 🙌 
We add UT only for units with logic.
```suggestion
```

Reply:
I only put it up for coverage. 
I deleted it.